stencil-library/src/components/details/utils.tsx

```diff
@@ -33,7 +33,7 @@ export const DetailSection = (
   <div part="detail-section" class="mt-4">
     <h5 part="detail-section-title">{sectionTitle}</h5>
     <hr />
-    <div class="d-flex flex-column justify-content-center gap-2 text-nowrap">
```

Comment:
Can you try making using some of these utility classes to make the key/value pair sizing behave like an HTML table?
https://getbootstrap.com/docs/5.3/utilities/display/

So this line would be `.d-table`, the children would be `.d-table-row`, and their children (ln51 and ln54) would be `.d-table-cell`.

I think that will fix some of the weird alignment issues with the key/value pairs (you may need to import those utility classes)